util/zenfs.cc

```diff
@@ -36,36 +37,39 @@ DEFINE_string(backup_path, "", "Path to backup files");
 
 namespace ROCKSDB_NAMESPACE {
 
-ZonedBlockDevice *zbd_open(bool readonly, bool exclusive) {
-  ZonedBlockDevice *zbd = new ZonedBlockDevice(FLAGS_zbd, nullptr);
+using ZonedBlockDevicePtr = std::unique_ptr<ZonedBlockDevice>;
```

Comment:
Please don't introduce new types unless it is really needed, we want to stick with rocksdb coding style (and I think adding types just makes the code harder to read - just another level of indirection)